_maps/map_files/moonstation/moonstation.dmm

```diff
@@ -64627,14 +64621,14 @@
 /area/station/security/brig/entrance)
 "rTb" = (
 /obj/effect/turf_decal/siding/dark,
-/obj/machinery/materials_market,
```

Comment:
see here for relevant code change since mapmerge likes to re-organise everything